### ISSUE TITLE
[Accessibility issue] Incorrect aria-label for add query on explore page (MAS ID: 256132a)

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#73743]
**Steps to reproduce**:
Turn on screen reader. Go to [`/explore`](https://play.grafana.org/explore) and navigate to the 'Add query' button. 

**Actual Result**:
The screen reader announces 'Add row button'.

**Expected Result**
The screen reader should announce 'Add query'.

![image](https://github.com/grafana/grafana/assets/6906380/9f308f4c-728c-4cf1-8933-b904e753786f)

**Comment by Elfo404** [comment]
this should have been fixed already by https://github.com/grafana/grafana/pull/73605 (to be released in 10.2).

I'll close this, feel free to reopen if i missed something
